pennylane/ops/qubit/special_unitary.py

```diff
@@ -279,6 +279,11 @@ class SpecialUnitary(Operation):
     differentiation techniques if the JAX, Torch or TensorFlow interface is used.
     See the theoretical background section below for details.
 
+    .. seealso::
+
+            For more details on using this operator in applications, see the
+            :doc:`in-depth demonstration <demos/tutorial_here_comes_the_sun>`.
```

Comment:
The links are still not working :thinking:

Reply:
Ok, this should be resolved shortly, it's currently broken due to infrastructure updates.

Reply:
Oh sorry! I forgot about this and resolved your original comment. I'll move it up